data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/OpenSearchSink.java

```diff
@@ -51,6 +51,7 @@ public class OpenSearchSink extends AbstractSink<Record<Event>> {
   public static final String BULKREQUEST_LATENCY = "bulkRequestLatency";
   public static final String BULKREQUEST_ERRORS = "bulkRequestErrors";
   public static final String BULKREQUEST_SIZE_BYTES = "bulkRequestSizeBytes";
+  public static final String DYNAMIC_INDEX_DROPPED_EVENTS = "DynamicIndexDroppedEvents";
```

Comment:
nit: camelCase

While it is a nit, I believe this is the value populated in the metrics store. We should be consistent. I am fine to address this later

Reply:
Addressed the comment. But for some reason end to end test is failing.